pyaerocom/ungriddeddata.py

```
@@ -508,173 +437,16 @@ def copy(self):
         from copy import deepcopy
```

Mid file imports [are against PEP8](https://peps.python.org/pep-0008/#imports) and they prevent import errors from failing early making refactoring difficult.

---

This is old code. Won't fix.

---

I understand not fixing all of these things in this PR, but I do think that it's good we make note of them in another issue and address them after this PR is merged. Can be for next milestone.